### PR TITLE
[seeder] Fix: lint standards

### PR DESCRIPTION
**Pull request by moonlitgrace**
`npm run lint`

### CODE REVIEW

File: seeder/src/main/interfaces/shiinobi.ts [hunk 1]
```diff
@@ -30,8 +30,8 @@ class Shiinobi {
 		}
 	}
 
-	#spawn({ command, id }: { command: _COMMANDS; id?: number }) {
-		let _command: string[] = [command];
+	#spawn({ command, id }: { command: _COMMANDS; id?: number }): Promise<unknown> {
+		const _command: string[] = [command];
```
Comment (moonlitgrace) [inline seeder/src/main/interfaces/shiinobi.ts:34]: @baseplate-admin do you agree with this change?
well this is eslint auto fixed one

Comment (baseplate-admin) [reply seeder/src/main/interfaces/shiinobi.ts:34]: well, this is not something we need.

Adding implicit return types

File: seeder/src/main/interfaces/shiinobi.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ class Shiinobi {
 		});
 	}
 
-	public async get_myanimelist_staff_urls() {
+	public async get_myanimelist_staff_urls(): Promise<unknown> {
```
Comment (baseplate-admin) [inline seeder/src/main/interfaces/shiinobi.ts:70]: having unknown promise

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: seeder/src/main/interfaces/shiinobi.ts (hunk 2 above)
Comment (moonlitgrace) [reply seeder/src/main/interfaces/shiinobi.ts:70]: Kinda cool right?

**Comment by moonlitgrace** [comment]
eslint failed

**Pull request closed without merging**